datatrails_scitt_samples/scripts/verify_receipt.py

```diff
@@ -132,10 +145,13 @@ def main():
         verified = verify_transparent_statement(transparent_statement, leaf)
 
     if verified:
-        print("signature verification succeeded")
-    else:
-        print("signature verification failed")
+        print("verification succeeded")
+        return True
+    print("verification failed")
+    return False
 
 
 if __name__ == "__main__":
-    main()
+    if not main():
```

Comment:
why not make main() return an int (0 or 1) then this could be one line

Reply:
sys.exit(main())

Reply:
that is what I originaly had. I thought the objection would be the otherway round - that returning ints from a "function" was fussy. I will switch it back

Reply:
note this means verified == 0 is *success* (i am fine with that)